app/services/processors/mappings.rb

```diff
@@ -34,8 +36,8 @@ def self.create(specification, user)
     # @return [Array]
     ###
     def self.create_mapping_terms mapping
-      spine = mapping.spine
-      spine.terms.each do |term|
+      terms = 0
+      mapping.spine.terms.each do |term|
         # Do not create this term if there's already one with the same uri
         next if MappingTerm.find_by(uri: term.desm_uri)
```

Comment:
Better use `exists?` instead of `find_by`, since there's no need to instantiate a term.